app/models/destination.rb

```diff
@@ -43,6 +43,11 @@ def detached_objects
     Detached.new(
       service: service,
       main_flow_uuids: grid.flow_uuids
-    ).flow_objects
+    )
+  end
+
+  def detached_flow_objects
```

Comment:
how does this differ from `flow_objects` method on the `Detached` model? they sound effectively the same, but perhaps there is a difference? if so then their names should not be so similar. i would be tempted to ask can you not call `detached_objects.flow_objects` on line 33, but perhaps i am missing a nuance

Reply:
Good question, calling `flow_objects` on the `Detached` model doesn't order them correctly, which is why we are currently seeing the drop down list out of sync with the Unconnected page section. The `detached_flows` method on the `Detached` model does this ordering for me. However, I am unable to call `detached_flows` on `detached_objects.flow_objects` as this returns an array.
I agree the naming is confusing, I can rename the method to `ordered_detached_objects`?